public/javascripts/Gallery/src/filter/ValidationOptionBucket.js

```diff
@@ -66,12 +66,21 @@ function ValidationOptionBucket(initialValidationOptions) {
         return bucket.filter(valOption => valOption.getActive()).map(valOption => valOption.getValidationOption());
     }
 
+    /**
+     * Resets the validation options to the default/initial values.
+     */
+    function setToDefault() {
```

Comment:
`setToDefault()` should do the job of unapplying the other severity options. As of now, to _actually_ set to the default, you are required to call the `unapplyValidationOptions()` function first, which isn't great, and differs from what you did for the label type menu!